src/status_im/chat/events/input.cljs

```diff
@@ -233,6 +234,31 @@
                                                   (min (count input-text)))]
                 (merge fx (update-text-selection new-db new-selection)))))))
 
+;; TODO(goranjovic) - generalize setting something as a command argument
+(defn set-asset-as-command-argument
+  "Sets asset as command argument for active chat"
+  [db {:keys [bot-db-key asset arg-index]}]
+  (let [name          (string/replace (name (:symbol asset)) (re-pattern constants/arg-wrapping-char) "")
+        command-owner (get-in (input-model/selected-chat-command db) [:command :owner-id])]
+    (-> db
+        (set-command-argument arg-index name true)
+        (bots-events/set-in-bot-db {:bot   command-owner
+                                    :path  [:public (keyword bot-db-key)]
+                                    :value asset})
+        (as-> fx
+              (let [{:keys [current-chat-id]
+                     :as new-db}             (:db fx)
+                    arg-position             (input-model/argument-position new-db)
+                    input-text               (get-in new-db [:chats current-chat-id :input-text])
+                    command-args             (cond-> (input-model/split-command-args input-text)
+                                               (input-model/text-ends-with-space? input-text) (conj ""))
```

Comment:
What is this conj  about?

Reply:
Don't ask me about this function or the one above. I've no idea how it works.